docs/source/introduction/index.rst

```diff
@@ -0,0 +1,163 @@
+MLflow 101
```

Comment:
Can we call this `What is MLflow?`. This is better for SEO and friendlier for non-english speakers.

I.e. instead of removing the existing page, can we update it with the content you've written here?

Reply:
For SEO purpose, shall we also add meta tags ? https://docs.readthedocs.io/en/stable/guides/technical-docs-seo-guide.html#use-meta-tags It helps search engine displays the website abstract better.